src/open_inwoner/openzaak/admin.py

```diff
@@ -18,23 +18,20 @@
     ZaakTypeInformatieObjectTypeConfig,
     ZaakTypeResultaatTypeConfig,
     ZaakTypeStatusTypeConfig,
+    ZGWApiGroupConfig,
 )
 
 
+class ZGWApiGroupConfig(admin.TabularInline):
```

Comment:
Let's use `StackedInline` here to avoid overflow to the right:

![image](https://github.com/maykinmedia/open-inwoner/assets/29249171/cb6e5415-50c9-439b-a829-b59071f89017)